checkov/common/checks_infra/solvers/attribute_solvers/jsonpath_not_equals_attribute_solver.py

```diff
@@ -0,0 +1,22 @@
+from typing import List, Optional, Any, Dict
+
+from checkov.common.checks_infra.solvers.attribute_solvers.jsonpath_equals_attribute_solver import JsonpathEqualsAttributeSolver
+from checkov.common.graph.checks_infra.enums import Operators
+
+
+class JsonpathNotEqualsAttributeSolver(JsonpathEqualsAttributeSolver):
+    operator = Operators.JSONPATH_NOT_EQUALS
+
+    def __init__(self, resource_types: List[str], attribute: Optional[str], value: Any) -> None:
+        super().__init__(resource_types=resource_types, attribute=attribute, value=value)
+
+    def get_operation(self, vertex: Dict[str, Any]) -> bool:
+        if self.attribute:
+            attribute_matches = self._get_attribute_matches(vertex)
+            if len(attribute_matches) == 0:  # the jsonpath is not found, so the check passes
```

Comment:
```suggestion
            if not attribute_matches:  # the jsonpath is not found, so the check passes
```
it is faster to check for falseness instead of calculating the length and check if it is 0. Both give the same result